app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,19 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Control Almacen</title>
+
+    <nav class="navbar navbar-dark navbar-expand bg-dark flex-md-nowrap p-0 shadow">
+      <ul class="navbar-nav px-3">
+        <li class="nav-item text-nowrap">
+          <%= link_to("log out", destroy_admin_session_path, method: :delete , class:'nav-link') %>
+        </li>
+        <li class="nav-item text-nowrap">
+        <a class="nav-link text-wrap" href="#">Profile</i></a>
+        </li>
+      </ul>
+      <a class="navbar-brand col-sm-3 col-md-2 ml-auto" href="#">Warehouse Control</a>
+    </nav>
```

Comment:
this is curious, I mean inside the `<head>` tag. I am used to seeing the `nav` element into the `body` or into `header`, not into the `head`

Reply:
yeah thats right, semantically this is not correct, this should be placed in the `body` tag